### ISSUE TITLE
Proper error wrapping

### ISSUE THREAD

**Issue by refs** [cs3org/reva#1075]
context:

> While working on this we realized we're using errors.Wrap all over the place on owncloud.go. GetMD(). I believe we should wrap on the lowest layers, in the context of this PR that would be getPath() and unwrapping on the top layers to assert on the underlying error type.

more context:

https://github.com/cs3org/reva/pull/1064

**Comment by labkode** [comment]
We should also take the opportunity to use the std new error library:
https://golang.org/pkg/errors/

**Comment by refs** [comment]
@labkode YES PLEASE. I was on talks about that with Jorn and Vincent yesterday. Dave's Chenney spoke his mind in moving away from gh/pkg/errors to the stdlib.

**Comment by refs** [comment]
@labkode does it make sense to add `technical debt` label to the repo, to capture all this?

**Comment by labkode** [comment]
Done
